main/src/library/FileWrite.flix

```diff
@@ -117,7 +117,7 @@ mod FileWrite {
             def write(data, f, k) = {
                 try {
                     let bytes = data#str.getBytes(StandardCharsets.UTF_8);
-                    Files.write(Paths.get(f), bytes, (...{}: Vector[OpenOption]));
+                    discard Files.write(Paths.get(f), bytes, (...{}: Vector[OpenOption]));
```

Comment:
@magnus-madsen I think this might also just be wrong, which is highlighted by the discard requirement. This PR does require some typing, but overall, it's not that much, and it's very good at finding errors.

Reply:
Not looking at the tests, it costs 26-36 `discard` to find 2-6 bugs, depending on how you count. Seems like a pretty good ratio to me